app/controllers/api/v1/tokens_controller.rb

```diff
@@ -3,23 +3,14 @@
 module API
   module V1
     class TokensController < Doorkeeper::TokensController
-      # Override Doorkeeper::TokensController#revoke to remove client credentials check
-      # We didn't use any client credentials
-      def revoke
-        # The authorization server responds with HTTP status code 200 if the client
-        # submitted an invalid token or the token has been revoked successfully.
-        if token.blank?
-          render json: {}
-          # The authorization server validates [...] and whether the token
-          # was issued to the client making the revocation request. If this
-          # validation fails, the request is refused and the client is informed
-          # of the error by the authorization server as described below.
-        elsif authorized?
-          revoke_token
-          render json: {}
-        else
-          render json: revocation_error_response, status: :forbidden
-        end
+      include API::V1::ErrorHandler
+
+      private
+
+      def revocation_error_response
+        {
+          errors: build_errors(detail: I18n.t('doorkeeper.errors.messages.revoke.unauthorized'), code: :unauthorized_client)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/700447c0a197333d7c81d82b62cd48614d8bc4d0/app/controllers/api/v1/tokens_controller.rb#L12"></span>Line is too long. [124/120]</td>
    </tr>
  </tbody>
</table>